### ISSUE TITLE
NFR: Accessible User Interface

### ISSUE THREAD

**Issue by tunajim** [kolton-64/MechGameSpring2025#32]
The user interface will follow accessibility standards established in the Web Content Accessibility Guidelines.  Buttons should be no smaller than 24x24.  Text should have a contrast ratio of 4.5:1.
